src/content/docs/dropins/all/extending.mdx

```diff
@@ -371,13 +371,19 @@ The following steps guide you through adding a third-party component to a drop-i
 <Task>
 ### Add your third-party API key
 
-The boilerplate starter code contains three config files, one for each environment: `configs.xlsx` (prod), `configs-stage.xlsx` and `configs-dev.xlsx` (or the equivalent Google sheets files). Add your third-party API key to the config environments you want to test or develop against. For example, if you're working in the dev environment, add the key to `configs-dev.xlsx` (or the equivalent Google sheet).
-
-| Key                      | Value                      |
-| ------------------------ | -------------------------- |
-| third-party-api-key      | third-party-api-value      |
-
-Replace `third-party-api-key` and `third-party-api-value` with the actual API key name and value.
+Add your API key to your commerce configuration - either in a local config.json or public config in config service.
```

Comment:
For ACCS customers -- do they need the API key? and maybe remove reference to public config in config service until we have some better info on using it?